bitsail-connectors/connector-fake/src/main/java/com/bytedance/bitsail/connector/fake/source/FakeRowGenerator.java

```diff
@@ -159,4 +173,107 @@ private Object fakeRawValue(TypeInfo<?> typeInfo) {
     }
     throw new RuntimeException("Unsupported type " + typeInfo);
   }
+
+  @SuppressWarnings("checkstyle:MagicNumber")
+  private Object constantRawValue(TypeInfo<?> typeInfo, String constantValue) {
```

Comment:
It seems that the code is redundant. Why not combine fakeRawValue and constantRawValue into one function

Reply:
When I was writing, I also thought about whether to implement fakeRawValue and constantRawValue in one function, I thought fakeRawValue would be too long and hard to maintain for different logic mix together after combining, so I separated them, and the cost is that we got 2 functions that look similar

Reply:
After the merger, the code will be more concise, looking forward to your contribution @kyle-hawk

Reply:
okay, thx for the guidance,  I'll merge them

Reply:
@lichang-bd I have merged them. please take a look. thx